### PR TITLE
Add news about sunsetting Mambaforge

### PR DESCRIPTION
**Pull request by jaimergp**
PR Checklist:

- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] if you are adding a new page under `docs/` or `community/`, you have added it to the sidebar in the corresponding `_sidebar.json` file
- [X] put any other relevant information below

Comes from https://github.com/conda-forge/miniforge/issues/602

### CODE REVIEW

**Comment by jaimergp** [comment]
pre-commit.ci autofix

File: news/2024-07-29-sunsetting-mambaforge.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+A year later, we would like to formally sunset Mambaforge. All users are encouraged to switch to
+Miniforge. Remember they contain the same packages, so the only difference is the default
+installation directory.
+
+The deprecation process will roughly follow this calendar:
+
+- August 2024: The Mambaforge installers will warn about the deprecation and make the user wait 30
+  seconds before continuing.
+- October 2024: The Mambaforge installers will refuse to install in pre-specified dates.
```
Comment (beckermr) [inline news/2024-07-29-sunsetting-mambaforge.md:19]:
```suggestion
- October 2024: The Mambaforge installers will refuse to install during several pre-specified date ranges (i.e., ["brownouts"](https://en.wikipedia.org/wiki/Brownout_(software_engineering))) in order to encourage users to switch to Miniforge.
```

**Review by beckermr — APPROVED** [review]

File: news/2024-07-29-sunsetting-mambaforge.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+A year later, we would like to formally sunset Mambaforge. All users are encouraged to switch to
+Miniforge. Remember they contain the same packages, so the only difference is the default
+installation directory.
+
+The deprecation process will roughly follow this calendar:
+
+- August 2024: The Mambaforge installers will warn about the deprecation and make the user wait 30
+  seconds before continuing.
+- October 2024: The Mambaforge installers will refuse to install during several pre-specified date ranges (i.e., ["brownouts"](<https://en.wikipedia.org/wiki/Brownout_(software_engineering)>)) in order to encourage users to switch to Miniforge.
```
Comment (beckermr) [inline news/2024-07-29-sunsetting-mambaforge.md:19]:
```suggestion
- October 2024: The Mambaforge installers will refuse to install during several pre-specified date ranges (i.e., ["brownouts"](<https://en.wikipedia.org/wiki/Brownout_(software_engineering)>)) in order to encourage users to switch to Miniforge. These dates are
  - Every two weeks in October 2024 (2024-10-01, 2024-10-15)
  - Every ten days in November 2024 (2024-11-01, 2024-11-10, 2024-11-20, 2024-11-30)
  - Every five days in December 2024 (2024-12-05, 2024-12-10, 2024-12-15, 2024-12-20, 2024-12-25, 2024-12-30, 2024-12-31)
```

**Review by beckermr — APPROVED** [review]

**Comment by jaimergp** [comment]
Thanks @beckermr! This is scheduled to be published today. If I don't hear any other thoughts by EOD, I'll click Merge. Pinging @conda-forge/core and @conda-forge/miniforge for visibility.

**Review by h-vetinari — APPROVED** [review]
👍

**Review by jezdez — APPROVED** [review]

**Pull request merged**